src/main/java/com/intel/gkl/IntelGKLUtils.java

```diff
@@ -57,16 +56,16 @@
      * @return  true if the native library is supported and loaded, false otherwise
      */
     @Override
-
-
     public synchronized boolean load(File tempDir) {
 
         if (!NativeLibraryLoader.load(tempDir, NATIVE_LIBRARY_NAME)) {
+            logger.warn(String.format("Unable to load %s", NATIVE_LIBRARY_NAME));
             return false;
         }
         if (!initialized) {
             initialized = true;
         }
+        logger.info(String.format("Loading %s library successfully", NATIVE_LIBRARY_NAME));
```

Comment:
Please check if it's duplicate for [NativeLibraryLoader](https://github.com/Intel-HLS/GKL/blob/master/src/main/java/com/intel/gkl/NativeLibraryLoader.java#L97l). May be we can check if the exception can be handled at this location.

Reply:
Same as above